source/Color.h

```diff
@@ -61,6 +61,13 @@ namespace Duel6 {
             set(red, green, blue, alpha);
         }
 
+        void operator=(const Color & c){
```

Comment:
Assignment operator by mel vracet referenci na this. A je jeste potreba pridat copy constructor, jinak mi to hazi warning.